infrastructure/main.tf

```diff
@@ -76,29 +51,31 @@ module "api" {
 }
 
 module "db" {
-  source              = "git@github.com:hmcts/moj-module-postgres?ref=master"
-  product             = "rpe-${var.product}-db"
-  location            = "${var.location_api}"
-  env                 = "${var.env}"
-  database_name       = "draftstore"
-  postgresql_user     = "draftstore"
-  sku_name            = "GP_Gen5_2"
-  sku_tier            = "GeneralPurpose"
-  common_tags         = "${var.common_tags}"
+  source                = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product               = "rpe-${var.product}"
+  location              = "${var.location_api}"
+  env                   = "${var.env}"
+  database_name         = "draftstore"
+  postgresql_user       = "draftstore"
+  postgresql_version    = "10"
```

Comment:
will module changes keep the current db in place?

Reply:
no I renamed the database so it will get deleted and recreated
this database isn't in use in production